lib/spack/spack/cmd/env.py

```diff
@@ -54,6 +54,97 @@
 ]
 
 
+#
```

Comment:
IIRC it was because I wanted to reuse the `create` functions inside the `activate` command but since they were defined after I was getting an undefined function error.